src/lib/components/copyInput.svelte

```diff
@@ -37,7 +37,8 @@
                 on:mouseenter={() => setTimeout(() => (content = 'Click to copy'))}
                 use:tooltip={{
                     content,
-                    hideOnClick: false
+                    hideOnClick: false,
+                    appendTo: document.getElementById('tooltip-container')
```

Comment:
Nice find! I do think this is on the right track. Looking at the documentation, appendTo can be a function that takes a reference to the element itself, so we can change this to:

```suggestion
                    appendTo: (ref) => ref.parentElement.parentElement
```

Be sure to add a comment or commit message to explain why we need to do `ref.parentElement.parentElement`..